src/DynamicImage/widget/DynamicImage.js

```diff
@@ -20,12 +20,21 @@
 
 	File is best readable with tabwidth = 2;
 */
-dojo.provide("DynamicImage.widget.DynamicImage");
+define([
+    "dojo/_base/declare",
+    "mxui/widget/_WidgetBase",
+    "dijit/_TemplatedMixin",
+	"dojo/_base/lang",
+	"dojo/dom-style",

```

Comment:
make sure indents are correct for newly added code

Reply:
Done